plugins/experimental/stale_while_revalidate/stale_while_revalidate.c

```diff
@@ -63,11 +63,7 @@ typedef struct {
   char *effective_url;
   TSMBuffer buf;
   TSMLoc http_hdr_loc;
-  union {
-    struct sockaddr sa;
-    struct sockaddr_in sin;
-    struct sockaddr_in6 sin6;
-  } client_addr;
+  struct sockaddr client_addr; // ipv4 or ipv6
```

Comment:
sockaddr can *never* be smaller than the standard network protocol addresses.   

Only AF_UNIX (I think) needs to be a long string-length value.